test/e2e/test_deb.py

```diff
@@ -39,12 +41,33 @@ def deb_archive_with_cartridge(cartridge_cmd, tmpdir, project_with_cartridge, re
                 "stress\n" +
                 "neofetch < 25")
 
+    net_msg_max = 1024
+    my_param = 'something'
+
+    systemd_unit_params = os.path.join(tmpdir, "systemd-unit-params.yml")
+    with open(systemd_unit_params, "w") as f:
+        f.write(f"""
+                instance-env:
+                    net-msg-max: {net_msg_max}
+                    my-param: {my_param}
+                """)
+
+    expected_param_values = f"""
+                            {{
+                            net_msg_max="{net_msg_max}",
+                            my_param="{my_param}"
+                            }}
+                            """
+
+    patch_init_to_cartridge_argparse_params(project, expected_param_values)
```

Comment:
`patch_init_to_cartridge_argparse_params` -> `patch_init_to_check_passed_params`.

It's OK to hardcode params directly in the init file for this test. It's simpler and easier to understand.